examples/sha256.rs

```diff
@@ -153,7 +155,12 @@ enum Sha256Coproc<F: LurkField> {
 /// Run the example in this file with
 /// `cargo run --release --example sha256 1 f5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4b false`
```

Comment:
Please annotate with the command that would allow printing the TexRay layer to stdout, and recall that command + the output in the PR summary for reviewers to easily get the point of the PR.